contracts/upgradeable_contracts/BasicOmnibridge.sol

```diff
@@ -88,7 +95,7 @@ abstract contract BasicOmnibridge is
 
         require(isTokenRegistered(token));
 
-        _handleTokens(token, false, _recipient, _value);
+        _handleTokens(token, false, _recipient, _shiftValue(token, _value));
```

Comment:
Since it is a hard decision who should own the remainder when the decimal shift happens, consider to remove the functionality to work with tokens with different decimals. Moreover it will allow to safe some gas.